ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -3691,6 +3691,27 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-long-duration
     workflow: cucushift-installer-rehearse-vsphere-ipi-zones
+- as: vsphere-ipi-zones-multisubnets-external-lb-f28
+  cron: 57 20 25 * *
+  steps:
+    cluster_profile: vsphere-multizone
+    env:
+      E2E_RUN_TAGS: '@amd64 and @vsphere-ipi and @network-ovnkubernetes and not @fips'
+      TAG_VERSION: '@4.14'
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-vsphere-ipi-zones-multisubnets-external-lb
+- as: vsphere-ipi-zones-multisubnets-external-lb-f28-destructive
+  cron: 51 3 24 * *
+  steps:
+    cluster_profile: vsphere-multizone
+    env:
+      E2E_RUN_TAGS: '@amd64 and @vsphere-ipi and @network-ovnkubernetes and not @fips
+        and @destructive'
```

Comment:
Would you mind to remove `and @destructive` here, it's no longer needed, refer to https://github.com/openshift/release/pull/43005

Reply:
no prob.